composer.json

```diff
@@ -10,9 +10,11 @@
         }
     ] ,
     "minimum-stability": "dev" ,
+    "prefer-stable": true ,
     "require":           {
         "php":            ">=7.1" ,
         "ext-json":       "*" ,
+        "ext-mbstring": "*" ,
```

Comment:
The `mbstring` extension should be loaded when using this package because this package uses the `mb_` functions.